routing/road_graph_router.cpp

```diff
@@ -1,10 +1,11 @@
+#include "routing/road_graph_router.hpp"
 #include "routing/bicycle_directions.hpp"
 #include "routing/bicycle_model.hpp"
+#include "routing/car_model.hpp"
```

Comment:
It is sorted. And #include "routing/road_graph_router.hpp" should go in the beginning. Is not it so?